hack/test-integration.sh

```diff
@@ -5,4 +5,4 @@ set -euo pipefail
 base=$( dirname "${BASH_SOURCE[0]}")
 
 go run "${base}/test-prerequisites.go"
-TEST_INTEGRATION=1 go test ./... -test.run=^TestIntegration -args -alsologtostderr -v=5
\ No newline at end of file
+TEST_INTEGRATION=1 go test ./... -test.run=^TestIntegration
```

Comment:
This is a problem because now we can't debug integration logs.  You'll need to have integration explicitly set both alsologtostderr or v5 (although pulling in klog should be sufficient to get the args are back).

Reply:
how does one do this given klog?

Reply:
ok, hopefully adding an init function will make it work...